plugins/sinks/file/README.md

```diff
@@ -0,0 +1,22 @@
+# file
+
+Sinks metadata to a file in `json/yaml` format as per the config defined.
+
+## Usage
+
+```yaml
+sinks:
+    name: file
+    config:
+        path: "./dir/sample.yaml"
+```
+
+## Config Defination
+
+| Key | Value | Example | Description |  |
+| :-- | :---- | :------ | :---------- | :-- |
+|`path` | `string` | `./dir/sample.yaml` | path to output file, directory should exist, file should be either yaml or json  | *required*|
```

Comment:
Is the path relative to binary / home folder?

Reply:
It is relative to binary in test, but even an absolute file path w.r.t home should work

Reply:
maybe better to add some details about that in the readme @GrayFlash

Reply:
ok doing it